install/kubernetes/helm/istio/values-istio-one-namespace-trust-domain.yaml

```diff
@@ -0,0 +1,22 @@
+# This is used to generate istio.yaml used for deprecated CI/CD testing.
+global:
+  # controlPlaneMtls enabled. Will result in delays starting the pods while secrets are
+  # propagated, not recommended for tests.
+  controlPlaneSecurityEnabled: true
+
+  mtls:
+    # Default setting for service-to-service mtls. Can be set explicitly using
+    # destination rules or service annotations.
+    enabled: true
+
+  ## imagePullSecrets for all ServiceAccount. Must be set for any clustser configured with private docker registry.
+  # imagePullSecrets:
+  #   - name: "private-registry-key"
+
+  # Default is 10s second
+  refreshInterval: 1s
+
+  trustDomain: test.local
+
+istiotesting:
+  oneNameSpace: true
```

Comment:
do we need many of these since many are already in values.yaml?  you only need the difference in this file.

Reply:
All values we specify here, have different defaults in values.yaml:
- controlPlaneSecurityEnabled: false

Reply:
Acutally, can you just add the line `trustDomain: test.local` to `istio-one-namespace.yaml` and `istio-one-namespace-auth.yaml`? I suppose trustDomain is a native feature we can test in the base tests, we don't need an extra test to cover this. WDYT?

Reply:
@myidpt wouldn't that mean there is no test left for the default?

Reply:
I would expect we would want to test _no-auth_, _auth_ and _auth-trust-domain_. But running all base tests in all three variants sounds indeed a bit much. Changing this would be some more effort and out-of-scope for this PR, right?
Also someone would have to decide on which test to run in which scenario.

From what I have understood, those tests use a deprecated framework anyway. Does this mean they will be migrated?

Reply:
Sorry for the late reply. That sounds good to me.